include/issues.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+function getIssue($issueId){
+    $result= dbQuery('
+        SELECT *
+        FROM issue
+        WHERE issueId= :issue
+    ', array(
+        'issueId'=> $issueId
```

Comment:
Nitpick: twice in this function (and maybe elsewhere I've just missed) you don't put a space before the `=` but you put one after. This looks a bit unbalanced to me. I'd recommend putting a space on both sides (so `issueId = :issue` or `'issueId' => $issueId`) but it'd also be ok to leave the spaces out. I'd just try to be consisted on both sides of the operator.